src/contract.rs

```diff
@@ -147,13 +151,26 @@ mod tests {
     fn addition_test() {
         let mut deps = mock_dependencies(20, &[]);
 
-        let msg = HandleMsg { a: 10, b: 20 };
-        let env = mock_env(&deps.api, "creator", &coins(1000, "earth"));
+        let msg = InitMsg::Init {
+            seed_phrase: "test".to_string(),
+        };
+        let env = mock_env("creator", &coins(1000, "earth"));
+        // we can just call .unwrap() to assert this was a success
+        let res = init(&mut deps, env, msg);
+        match res.unwrap() {
+            InitResponse => {}
+        }
+
+        let msg = HandleMsg::NewKey {
+            key_seed: "test".to_string(),
+            passphrase: "test".to_string(),
+        };
+        let env = mock_env("creator", &coins(1000, "earth"));
         // we can just call .unwrap() to assert this was a success
         let res = handle(&mut deps, env, msg);
         match res {
-            Ok(resp) => println!(resp),
-            _ => assert_eq!(0, 1),
+            Ok(resp) => println!("{:?}", resp),
+            Err(err) => println!("{}", err),
         }
```

Comment:
in the original code, an `Err` would cause a test failure. in your case, it doesn't.
This is the idiomatic way of failing the test, in this case:
```rust
println!("{:?}", res.unwrap()))
```
An even better way would be to put `-> StdResult<()>` in the test function return type, which would allow you to write:
```rust
println!("{:?}", res?);
```
instead of the `match` expression